slim_trees/sklearn_tree.py

```diff
@@ -46,7 +46,7 @@ def dumps(model: Any) -> bytes:
 def _tree_pickle(tree: Tree):
     assert isinstance(tree, Tree)
     reconstructor, args, state = tree.__reduce__()
-    compressed_state = _compress_tree_state(state)
+    compressed_state = _compress_tree_state(state) # type: ignore
```

Comment:
can you specify which mypy check fails here?

Reply:
Pylance is failing